src/fare-contracts/PurchaseReservation.tsx

```diff
@@ -42,9 +40,6 @@ export const PurchaseReservation: React.FC<Props> = ({reservation}) => {
     }
   };
 
-  // Filter out reservations for subaccounts
-  if (reservation.customerAccountId !== abtCustomerId) return null;
```

Comment:
Originally this prevented reservations for subaccounts to be shown. After the changes introduced here, those reservations will now show in `sent` mode. Is this what's desired?

Observation: we have `fareContracts` and `sentFareContracts`, but no such distinction for `reservations` or `rejectedReservations`.

Reply:
Right now, with the new filtering, the subaccount reservations should be shown on tickets sent for others. Wouldn't you think it is enough? or should we separate it as well?

Reply:
I think it's probably enough.
However @Sebstorvik @mjenssen you might have some thoughts about this:
- Should tickets bought for subaccounts be shown in ticket history sent to others? If so, should they look exactly the same as any other ticket bought on-behalf-of?
- Technically, should we also have `sentReservations` and `sentRejectedReservations`?

Reply:
IMO, I think the purchase history should always be visible for the purchaser, no matter who gets the ticket in the end. The subaccount system was the initial on-behalf-of solution, so I think the faster we merge these two solutions the better.

As for splitting reservations, I think it's fairly easy to filter what's in that collection. Unless someone has a convincing argument for splitting, I propose we keep it as it is for now. (The `sentFareContracts` collection was added to prevent old apps from showing sent fare contracts in the main view.)

Long-term, we should look at data modeling of the whole reservation / orders / fare contracts, as well as how this could look in a non-Firestore world.

Reply:
Sounds good to me, just want to be sure we are making conscious decisions here :+1:
The split only matters if we need the ability to separate tickets bought on-behalf-of and for a sub-account for some reason, but I guess there is no such reason as of today.

Reply:
> The subaccount system was the initial on-behalf-of solution, so I think the faster we merge these two solutions the better.

I couldn't agree more. But that's for 2025, I guess..